src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -269,7 +269,7 @@ private void validateColumnType(Errors errors, ColumnModel columnModel, String k
     }
 
     Set<TableDataType> invalidTypes = Set.of(TableDataType.DIRREF, TableDataType.FILEREF);
-    if (invalidTypes.contains(columnModel.getDatatype())) {
+    if (columnModel.getDatatype() != null && invalidTypes.contains(columnModel.getDatatype())) {
```

Comment:
Should we instead have a separate check that checks to make sure that datatypes aren't null?  Or do we have that but it's _also_ doing this check?  Just want to make sure that if the user doesn't provide a datatype, we return an error explicitly about that

Reply:
Yeah we check the datatypes when validating the table [here](https://github.com/DataBiosphere/jade-data-repo/blob/8a36c374c4b4a50f8851534d07c2ac1d5f818da8/src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java#L290), so in this case we still return those errors!

Reply:
Ah perfect, thanks!